happi/backends/core.py

```diff
@@ -1,24 +1,22 @@
 """
-Abstract backend database options
+Base backend database options
 """
-import abc
 import logging
 
 logger = logging.getLogger(__name__)
 
 
-class Backend(abc.ABCMeta):
+class _Backend:
     """
-    Abstract interface for backend database
+    Base class for backend database
     """
-    @abc.abstractproperty
-    def devices(self):
+
+    def all_devices(self):
```

Comment:
Let's keep this a `@property` otherwise it's callable now instead

Reply:
It was deleted by accident. Addressed via https://github.com/pcdshub/happi/pull/115/commits/e44f98f7a5daeac13ef52c1a07c66d907260eb2e